CoronaContact/Logic/QuarantineTimeController.swift

```diff
@@ -204,7 +221,9 @@ class QuarantineTimeController {
             endOfQuarantines.append(endOfQuarantine)
         }
 
-        let sortedQuarantines = endOfQuarantines.sorted { $0.date.compare($1.date) == .orderedDescending }
+        let sortedQuarantines = endOfQuarantines.sorted {
+            $0.date.compare($1.date) == .orderedDescending
```

Comment:
I'm not sure but I think this would be enough:
```suggestion
            $0.date > $1.date
```